dockerfiles/Dockerfile.arch-freeze

```diff
@@ -0,0 +1,47 @@
+FROM scratch
+LABEL maintainer="Iterait a.s. <docker@iterait.com>"
+
+COPY --from=archlinux:latest / /
+# add cleanup hook
+ADD 99-cleanup.hook /usr/share/libalpm/hooks/
+
+
+# freeze Arch to specific date
+ARG FREEZE_DATE='2024/04/07'
+
+RUN sed -i '/^\[core\]$/,/^$/c\[core]\nSigLevel = PackageRequired\nServer=https://archive.archlinux.org/repos/'$FREEZE_DATE'/$repo/os/$arch\n' /etc/pacman.conf
```

Comment:
what kind of `'` escaping is this? is this intentional?

Reply:
Yes, it is, first you have the 'first part of sed' escaped, then docker will append $FREEZE_DATE and then append fully escaped second part of the sed